docs/data-sources/custom_qos_policy.md

```diff
@@ -53,11 +53,11 @@ data "aci_custom_qos_policy" "example_tenant" {
 * `name_alias` (nameAlias) - (string) The name alias of the Custom Qos Policy object.
 * `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
 * `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
-
+  
```

Comment:
why the spaces?

Reply:
This only appears in the code and not in the actual docs. There are removable though